src/main/java/com/salesforce/dataloader/dyna/SforceDynaBean.java

```diff
@@ -313,7 +313,7 @@ synchronized static public void registerConverters(Config cfg) {
         // Register DynaBean type conversions
         ConvertUtils.register(new DateConverter(tz, useEuroDates), Calendar.class);
         ConvertUtils.register(new DoubleConverter(), Double.class);
-        ConvertUtils.register(new IntegerConverter(), Integer.class);
+        ConvertUtils.register(new IntegerConverter(null), Integer.class);
```

Comment:
Do we want to set default as null or 0? Maybe I am not fully understanding the problem here...